api/views.py

```diff
@@ -33,21 +33,16 @@ class UserViewSet(ModelViewSet):
     http_method_names = ['get', 'post']
 
 
-class AssetViewSet(ModelViewSet):
+class ManageAssetViewSet(ModelViewSet):
     serializer_class = AssetSerializer
-    permission_classes = [IsAuthenticated, ]
+    permission_classes = [IsAuthenticated, IsAdminUser]
     authentication_classes = (FirebaseTokenAuthentication,)
-    http_method_names = ['get', 'post']
+    http_method_names = ['get', 'post', 'put', 'delete']
 
     def get_queryset(self):
-        user = self.request.user
-        is_admin = self.request.user.is_staff
         queryset = Asset.objects.all()
         query_params = self.request.query_params
 
-        if not is_admin:
-            queryset = Asset.objects.filter(assigned_to=user)
-
         if query_params.get('email'):
```

Comment:
Let's move this to the `AssetViewSet ` as well

Reply:
Done!